source/educators/how-tos/add_custom_css_or_javascript_code_to_a_course.rst

```diff
@@ -0,0 +1,43 @@
+.. _Add Custom CSS or JavaScript Code to a Course:
+
+Add Custom CSS or JavaScript Code to a Course
+#############################################
+
+.. tags:: educator, how-to
+
+#. In Studio, from the **Settings** menu, select **Advanced Settings**.
+
+#. Locate the policy keys for **Course-wide custom css** and **Course-wide custom js**.
+   The default values will be an empty list ``[]``.
+
+#. You can add URLs to any CSS or JavaScript files in their respective fields. You can use
+   absolute or relative URLs, as you can see in the following example:
+
+   .. code:: json
+
+      [
+         "https://some.cdn.com/library.js",
+         "/asset-v1:OpenedX+DemoX+DemoCourse+type@asset+block@some-script.js"
+      ]
+
+#. You can upload scripts and CSS files to a course though the **Files** page accessible
+   under the **Content** menu in Studio.
+
+.. note::
+
+   Custom CSS or JavaScript added to a course in this manner will only be
+   available in the course content and will not apply to the rest of the
+   site's UI.
+
+.. seealso::
+
+   :ref:`Add Course Files` (how-to)
+
+
+**Maintenance chart**
+
++--------------+-------------------------------+----------------+--------------------------------+
+| Review Date  | Working Group Reviewer        |   Release      |Test situation                  |
++--------------+-------------------------------+----------------+--------------------------------+
+|              |                               |                |                                |
```

Comment:
Can you fill out this table with today's date and what release these instructions were tested with? The test situation will be "Pass"

Reply:
@xitij2000 For my test I used the master branch.

Reply:
I think you could say Teak, then.

Reply:
I see the config option in Redwood as well, but I haven't actively tested it on that release. I can test it there if having an earlier release would be better.

Reply:
Ideally we actually want these docs to be up-to-date with the latest release. If you think there's a chance it works on master but *not* on Redwood it may be worth testing it there but if the codepaths haven't changed I think just saying this will work on the Teak release (and assumed before that as well) will be good.

If you do test on Redwood, I'd maybe actually make two entries in this table - one for Redwood and one for Teak.

Reply:
@sarina I've filled in the table, and tagged you as the reviewer. Do tell if it should be the full name instead or if it should be someone else.

Reply:
Generally it's the document author but this is fine this time.